xs2a-adapter-aspsp-registry/src/test/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package de.adorsys.xs2a.adapter.registry;
+
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import de.adorsys.xs2a.adapter.registry.mapper.AspspMapper;
+import de.adorsys.xs2a.adapter.service.AspspCsvService;
+import org.apache.lucene.store.ByteBuffersDirectory;
+import org.junit.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AspspCsvServiceImplTest {
+
+    private LuceneAspspRepository luceneAspspRepository = new LuceneAspspRepository(new ByteBuffersDirectory());
```

Comment:
If I were you, I would mock `LuceneAspspRepository` instance and afterward mock its behavior (`luceneAspspRepository.findAll()`)